### PR TITLE
Added example usages of RangeWithOptions, MultiRangeWithOptions, and MultiGetWithOptions

### PR DESCRIPTION
**Pull request by filipecosta90**
Added examples to the godoc regarding RangeWithOptions, MultiRangeWithOptions, and MultiGetWithOptions methods.

### CODE REVIEW

File: example_client_test.go [hunk 1]
```diff
@@ -18,5 +18,87 @@ func ExampleNewClientFromPool() {
...
+// Exemplifies the usage of MultiRangeWithOptions function.
+func ExampleMultiRangeWithOptions() {
+	host := "localhost:6379"
+	password := ""
+	pool := &redis.Pool{Dial: func() (redis.Conn, error) {
+		return redis.Dial("tcp", host, redis.DialPassword(password))
+	}}
+	client := redistimeseries.NewClientFromPool(pool, "ts-client-1")
+
+	labels := map[string]string{
```
Comment (ashtul) [inline example_client_test.go:50]: labels1

File: example_client_test.go [hunk 2]
```diff
@@ -18,5 +18,87 @@ func ExampleNewClientFromPool() {
...
+	}
+	client.AddWithOptions("time-serie-1", 2, 1.0, redistimeseries.CreateOptions{RetentionMSecs: 0, Labels: labels})
+	client.Add("time-serie-1", 4, 2.0)
+
+	labels2 := map[string]string{
+		"machine": "machine-2",
+		"az":      "us-east-1",
+	}
+	client.AddWithOptions("time-serie-2", 1, 5.0, redistimeseries.CreateOptions{RetentionMSecs: 0, Labels: labels2})
+	client.Add("time-serie-1", 4, 2.0)
```
Comment (ashtul) [inline example_client_test.go:62]: ??

File: example_client_test.go [hunk 3]
```diff
@@ -18,5 +18,87 @@ func ExampleNewClientFromPool() {
...
+// Exemplifies the usage of MultiGetWithOptions function while using the default MultiGetOptions and while using user defined MultiGetOptions.
+func ExampleMultiGetWithOptions() {
+	host := "localhost:6379"
+	password := ""
+	pool := &redis.Pool{Dial: func() (redis.Conn, error) {
+		return redis.Dial("tcp", host, redis.DialPassword(password))
+	}}
+	client := redistimeseries.NewClientFromPool(pool, "ts-client-1")
+
+	labels := map[string]string{
```
Comment (ashtul) [inline example_client_test.go:80]: same

File: example_client_test.go [hunk 4]
```diff
@@ -18,5 +18,87 @@ func ExampleNewClientFromPool() {
...
+	}
+	client.AddWithOptions("time-serie-1", 2, 1.0, redistimeseries.CreateOptions{RetentionMSecs: 0, Labels: labels})
+	client.Add("time-serie-1", 4, 2.0)
+
+	labels2 := map[string]string{
+		"machine": "machine-2",
+		"az":      "us-east-1",
+	}
+	client.AddWithOptions("time-serie-2", 1, 5.0, redistimeseries.CreateOptions{RetentionMSecs: 0, Labels: labels2})
+	client.Add("time-serie-1", 4, 2.0)
```
Comment (ashtul) [inline example_client_test.go:92]: same

Comment (filipecosta90) [reply example_client_test.go]: fixed "time-serie-1" to "time-serie-2"

**Comment by filipecosta90** [comment]
> looks good. minor comments

fixed the the ts.add with "time-serie-1" to "time-serie-2" on the last addition on the examples. Also changes labels to labels1.

**Pull request merged**